_posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd

```diff
@@ -0,0 +1,130 @@
+  Reflecting on 2022.
+author:
+  - name: Chloe Pou-Prom, Meggie Debnath, Derek Beaton, Cathy Zhu
+date: 2022-12-13
+output:
+  distill::distill_article:
+    self_contained: false
+draft: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+The end of the year is a time for reflection, reminiscing, and sharing screenshots of our Spotify Wrapped. For those who are unfamiliar, Spotify Wrapped is a yearly marketing campaign by Spotify, where they compile streaming data and create a personalized graphic for each user. ^[We surveyed the members on DSAA and found that the most listened music genre is _Pop_, followed by various interesting variations of _Indie_ (indie pop, indie soul, indie Québécois).] 
+
+In the spirit of data, end-of-year reflections, and colorful infographics, we thought it would be fun to create our very own DSAA wrapped!
+
+![](dsaa_wrapped.PNG)
+
+Enjoy some stats!!
+
+# Projects
+
+2022 was a busy year with lots of projects!!
+
+- Total \# of projects deployed: 9
+
+- Most viewed project: Ops Centre
+
+    - The Ops Centre project provides a clear picture of patient flow within the hospital. It consists of multiple data tiles that reflect real-time data in the areas of occupancy, upcoming discharges, planned admissions, bed cleaning, portering, and other elements essential to providing a full-picture of patient flow.
+    
+- Project with the most team members whose names start with _M_: Hemodialysis
+
+    - The Hemodialysis project is a weekly email alert that gets sent to clinicians at the St. Michael's Hospital hemodialysis clinics. The goal of the project is to predict a patient's risk of a hospital visit in the 30 days following their dialysis sessions.
+    
+
+# Product families
+
+2022 was the year of _product families_.^[Blog post coming soon!! :fingers_crossed:]
+
+![](https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fimages.squarespace-cdn.com%2Fcontent%2Fv1%2F58b3cae5d1758e0132a008c6%2F1582078446179-VU0VAYNPOV4P3JIBKNIY%2FIt%2527s%2BAbout%2BFamily%2BLogo%2B1500.png%3Fformat%3D1000w)
+
+Whoa, wait. What exactly is a product family? 
+
+- **Product** at DSAA is a _platform, application, system, or report_ that is created to meet patient, clinician, or hospital operational needs.
+
+- **Product family** (roughly comparable to a product line) is a group of related products that are developed and maintained by a single cross-functional team at DSAA. 
+
+- We adopted this custom framework in order to allow staff to work effectively within persistent teams and to leverage resources across products the best we can. The products under each product family are related by common product value and/or shared domain expertise requirement.
+
+
+Some product family stats!
+
+- Number of product families: 6
+
+- Product family with the most projects: The _Patient Flow_ product family ^[Fun fact: The _Patient Flow_ product family is also the product family with the most Fast & Furious memes.]
```

Comment:
Cathy confirmed that there are 6 families and that Patient Flow has the most projects